contrib/zeromq/csharp/ThriftZMQ.csproj

```diff
@@ -25,7 +25,7 @@
     <UpdateRequired>false</UpdateRequired>
     <MapFileExtensions>true</MapFileExtensions>
     <ApplicationRevision>0</ApplicationRevision>
-    <ApplicationVersion>1.0.0.%2a</ApplicationVersion>
+    <ApplicationVersion>0.12.0</ApplicationVersion>
```

Comment:
I think you want to leave %2a, MAYBE... not sure, but it happened below.  That said, it might be a remnant and 0.12.0.0 would be more appropriate.